packages/next/build/webpack/plugins/build-manifest-plugin.ts

```diff
@@ -166,19 +150,6 @@ export default class BuildManifestPlugin {
               false
             )};` + `self.__BUILD_MANIFEST_CB && self.__BUILD_MANIFEST_CB()`
           )
-
-          if (this.modern) {
-            const modernClientManifestPath = `${CLIENT_STATIC_FILES_PATH}/${
-              this.buildId
-            }/_buildManifest.module.js`
-
-            compilation.assets[modernClientManifestPath] = new RawSource(
-              `self.__BUILD_MANIFEST = ${generateClientManifest(
-                assetMap,
-                true
-              )};` + `self.__BUILD_MANIFEST_CB && self.__BUILD_MANIFEST_CB()`
```

Comment:
Doesn't this code actually create the buildManifest asset? Is it ok to delete it without replacing with anything that does the same?

Reply:
This is only needed for the modern build, since Im loading the file explicitly, we don't need the `.module.js` variant